explorer/src/lib/services/duskAPI.js

```diff
@@ -81,6 +82,17 @@ const gqlGet = (queryInfo) =>
     .then(failureToRejection)
     .then((res) => res.json());
 
+/**
+ * @param {string} uri
+ */
+const ruesGet = (uri) =>
```

Comment:
I have added, essentially, the same function in my PR here: https://github.com/dusk-network/rusk/pull/2679/files#diff-3baa3a3c85f36dcce31a6c71251e6d4cc4b9e3b965825946f1e19c760250a4d1R109